SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -540,7 +540,7 @@ void GenerateSubobjectSchemaForActor(FComponentIdGenerator& IdGenerator, UClass*
 		{
 			UObject* Value = PropertyTypeInfo->Object;
 
-			if (Value != nullptr && !Value->IsEditorOnly())
+			if (Value != nullptr && !Value->IsEditorOnly() && Value->IsSupportedForNetworking())
```

Comment:
Can we really rely on this being valid in Editor?

Reply:
Yes I believe so. As far as I can tell this works.

Reply:
As discussed, figure out what exactly this filters out. If nothing, remove this.